code/datums/diseases/advance/symptoms/cockroach.dm

```diff
@@ -40,9 +40,18 @@
 			if(prob(50))
 				to_chat(M, "<span class='notice'>You feel in tune with the station</span>")
 		if(5)
-			if(prob(10))
+			if(prob(30))
 				M.visible_message("<span class='danger'>[M] squirms as a cockroach crawls from their pores!</span>", \
 								  "<span class='userdanger'>A cockroach crawls out of your face!!</span>")
 				new /mob/living/simple_animal/cockroach(M.loc)
 			if(prob(50))
 				to_chat(M, "<span class='notice'>You feel something crawling in your pipes!</span>")
+	
+/datum/symptom/cockroach/OnDeath(datum/disease/advance/A)
+	if(!..())
+		return
+	if(death_roaches)
+		var/mob/living/M = A.affected_mob
+		to_chat(M, "<span class='warning'>Your pores explode into a colony of roaches!</span>")
+		for(var/i in 1 to rand(1,5))new /mob/living/simple_animal/cockroach(M.loc)
```

Comment:
Christ.

Reply:
?

Reply:
if it's the last line you're worried about, that's literally the only way it could pass travis

Reply:
You put two lines of code on one line.

Reply:
only thing that travis didnt hate. i mean, ill try again...

Reply:
.... what the fuck, it's fine now?


some shit i'll never understand